test/CMakeLists.txt

```diff
@@ -196,6 +206,16 @@ itk_add_test(NAME itkSpectra1DNormalizeImageFilterTest
     DATA{Input/spectra_pixel_image.mha}
     ${ITK_TEST_OUTPUT_DIR}/itkSpectra1DNormalizeImageFilterTest.mha
     )
+itk_add_test(NAME itkSpectra1DNormalizeImageFilterTest2
+  COMMAND UltrasoundTestDriver
+    --compare
+    DATA{Baseline/itkSpectra1DNormalizeImageFilterTest2Baseline.mha}
+    ${ITK_TEST_OUTPUT_DIR}/itkSpectra1DNormalizeImageFilterTest2.mha
+  itkSpectra1DNormalizeImageFilterTest
+    DATA{Input/UnfusedRF-a0-spectra-cropped.mhd,UnfusedRF-a0-spectra-cropped.raw}
+    DATA{Input/UnfusedRF-a0-spectra-cropped.mhd,UnfusedRF-a0-spectra-cropped.raw}
```

Comment:
Why not use `DATA{Baseline/itkSpectra1DAveragingImageFilterTest2Baseline.mha}` as reference? That will produce meaningful, not-trivial output.

Reply:
Ah, good point. Will make the change.